### PR TITLE
Do not proxy kubernetes internal traffic

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Do not proxy kubernetes internal traffic

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
Fixes #914

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ sumologic:
   ## proxy urls
   httpProxy: ""
   httpsProxy: ""
+  ## Exclude kubernetes internal traffic from proxy
+  noProxy: kubernetes.default.svc
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml:36]: Love this approach it is cleaner.  Con is that if a customer has a different url where they customize the host in the TF kubernetes provider, they have to also update this.  Any way to have one property to control both?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:36]: Should the cluster endpoint be excluded from the proxy every time?
I can merge both of the config options in helm, but there will be no way to include the kubernetes url then

Comment (frankreno) [reply deploy/helm/sumologic/values.yaml:36]: Ahh ok fair point. In that case we can leave it

**Review by frankreno — APPROVED** [review]
Lgtm

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**